services/motion/builtin/builtin.go

```diff
@@ -237,7 +258,103 @@ func (ms *builtIn) MoveOnGlobe(
 	angularVelocity float64,
 	extra map[string]interface{},
 ) (bool, error) {
-	return false, ErrNotImplemented
+	operation.CancelOtherWithLabel(ctx, builtinOpLabel)
+
+	// create a new empty framesystem which we add our base to
+	fs := referenceframe.NewEmptyFrameSystem("")
+
+	// get the localizer from the componentName
+	localizer, ok := ms.localizer[movementSensorName]
+	if !ok {
+		return false, resource.DependencyNotFoundError(movementSensorName)
+	}
+
+	currentPIF, _ := localizer.CurrentPosition(ctx)
```

Comment:
Yeah I think this assumption is reasonable for now and the improvement is outside the scope of this current work.  Can you make a ticket to track this improvement Nick?

Reply:
ok this is a good callout, but I think this is currently not possible. Suppose I do not do specify a frame for a base in its attribute section.
to get the parent we would do:
```
	robotFS, err := ms.fsService.FrameSystem(ctx, nil)
	if err != nil {
		return false, err
	}
	
	localizerFrame := robotFS.Frame(movementSensorName.ShortName())
	localizerParent, err := robotFS.Parent(localizerFrame)
```
however here `localizerParent` will be nil

Reply:
>  Can you make a ticket to track this improvement Nick?

yea you got it!

https://viam.atlassian.net/browse/RSDK-3557

Reply:
The amount of time it would take the ticket, argue about whether we should do it in future scoping/triage meetings, get it backlogged, eventually have it become obvious that it's needed, dig it up, and then remember where to add the code, and do it, is significantly larger than the 45 seconds it would take to add the 12 lines of code it would take to do this, 6 of which Nick already wrote above.

For that reason I think we should do this as part of this PR.